### ISSUE TITLE
Video name

### ISSUE THREAD

**Issue by nicostubi** [Codeception/CodeceptJS#3402]
#### What are you trying to achieve?

Having a video for a failed test with the scenario name, as for screenshots.

#### What do you get instead?

File names with ids similar to "2f6507fa202d0d3e5d8cc2a55850a285.webm"

### Details

* CodeceptJS version: 3.3.5 
* NodeJS Version: 16.10.0
* Operating System: Win 10 Pro
* playwright
* Configuration file:

```js
require('ts-node/register')
const { setHeadlessWhen } = require('@codeceptjs/configure')

// turn on headless mode when running with HEADLESS=true environment variable
// HEADLESS=true npx codecept run
setHeadlessWhen(process.env.TEST_HEADLESS === 'true')

exports.config = {
	tests: './tests/*/**_test.ts',
	output: './tests/output',
	timeout: 240, // limit all tests in all suites to 240 secs
	helpers: {
		Playwright: {
			browser: 'chromium',
			show: true,
			url:'http://localhost:3030',
			video: true,
			trace: false,
			// keepTraceForPassedTests: true,
			waitForAction: 2000,
			waitForTimeout: 30000,
			windowSize: '1800x920',
		},
	},
	bootstrapAll: require('./presettings.ts').bootstrapAll,
	bootstrap: require('./presettings.ts').bootstrap,
	teardownAll: require('./presettings').teardownAll,
	teardown: require('./presettings').teardown,
	plugins: {
		subtitles: {
			enabled: true
		},
		pauseOnFail: {},
		retryFailedStep: {
			enabled: true,
		},
		tryTo: {
			enabled: true,
		},
		screenshotOnFail: {
			enabled: true,
		},
		// coverage: {
		// 	enabled: true,
		// },
	},
}

```

**Comment by nicostubi** [comment]
Hello, 

Any news? 

BR
Nicolas

**Comment by kobenguyent** [comment]
![Screen Shot 2022-09-29 at 10 17 18 AM](https://user-images.githubusercontent.com/7845001/192978648-d23f1986-4563-4c43-8749-f766e07133fe.png)
I suspect that name is from Playwright.

**Comment by nicostubi** [comment]
Hello @PeterNgTr 

I've just seen this has been released in version 3.3.6.

However, I still have video names looking like UIDs. 

Is there any configuration to apply ?

Best Regards,
Nicolas

**Comment by kobenguyent** [comment]
Hello @nicostubi not any configuration is needed.
